### PR TITLE
Minor additions/fixes to ingress operator docs

### PR DESCRIPTION
**Pull request by danehans**
- Explains that the default `ingresscontroller` will be auto recreated.
- Modifies the Ingress Operator intro in `configuring-ingress-operator.adoc`

### CODE REVIEW

**Comment by bmcelvee** [comment]
LGTM - thanks, @danehans!

@lihongan PTAL, thanks!

File: networking/ingress/configuring-ingress-operator.adoc [hunk 1]
```diff
@@ -5,15 +5,13 @@ include::modules/common-attributes.adoc[]
...
-Operator to route traffic by specifying OpenShift `Route` and Kubernetes `Ingress`
-resources.
-
-The Ingress Operator deploys and manages an OpenShift router, which is a
-HAProxy-based Kubernetes Ingress controller.
-
-Ingress Operator implements the OpenShift `ingresscontroller` API.
+The Ingress Operator implements the `ingresscontroller` API and is the
+component responsible for enabling external access to OpenShift
+cluster services. The Operator makes this possible by deploying and
```
Comment (ironcladlou) [inline networking/ingress/configuring-ingress-operator.adoc]: should Operator be capitalized here?

Comment (danehans) [reply networking/ingress/configuring-ingress-operator.adoc]: Thanks for catching this. I changed to lowercase and updated the PR.

**Comment by lihongan** [comment]
LGTM - thanks @danehans

**Pull request merged**